editor/property_editor.cpp

```diff
@@ -4287,6 +4284,11 @@ String PropertyEditor::get_selected_path() const {
 		return "";
 }
 
+bool PropertyEditor::get_capitalize_paths() const {
```

Comment:
Yes, I've given some though to that and the "is"-prefixed version also seemed weird to me.

Your suggestion seems right to me, so I'll edit & push.